T/Torch/deps.md

```diff
@@ -0,0 +1,76 @@
+# Dependencies
```

Comment:
What is this used for?  Apart from being a fancy dependency visualisation, that is

Reply:
That is for maintenance purposes: I have had good use of that visualisation in order to grasp all the dependencies - in order to not constantly check git submodules and cmake dependencies in cpuinfo, pthreadpool, xnnpack and PyTorch - there are even a few that's not visualised...